helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/MetadataStoreRoutingDataReader.java

```diff
@@ -35,8 +35,7 @@
    * @return a mapping from "metadata store realm addresses" to lists of "metadata store sharding
    *         keys", where the sharding keys in a value list all route to the realm address in the
    *         key
-   * @throws InvalidRoutingDataException - when the routing data is malformed in any way that
-   *           disallows a meaningful mapping to be returned
+   * @throws InvalidRoutingDataException - when the routing data is missing
```

Comment:
Could we be more specific about this? What does it actually mean to have the routing data "missing"? Which ZNodes do not exist? Which fields do not exist?

Reply:
Zk vocabulary shouldn't be used here because this is a generic interface. I can rework the wording here, maybe just reverting to the original one because it's still true.

Reply:
That's a good point. You could write that in "ZK"RoutingDataReader.

Reply:
Done.